.github/workflows/ci-all-in-one-build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     - uses: actions/setup-go@v2
       with:
-        go-version: ^1.17
+        go-version: ^1.17.6
```

Comment:
What version does the 1.17 pick up? I would assume it is the latest one?

Reply:
It probably does, but why risk it. 1.18 is supposed to come out in Feb iiuc, so we'll switch to that and won't have the patch no frozen.

Reply:
good